packages/core/src/generators/move/generator.ts

```diff
@@ -13,6 +13,7 @@ import {
 } from '@nrwl/devkit';
 import { dirname, extname, relative } from 'path';
 import { MoveGeneratorSchema } from './schema';
+import { getRelativePathToRootTsConfig } from '@nrwl/js';
```

Comment:
Lets avoid tight-coupling here, @nrwl/js (or @nx/js) should be an optional peer dep. I'd prefer we use: https://nx.dev/nx-api/devkit/documents/offsetFromRoot#function-offsetfromroot